### ISSUE TITLE
Add option to use actual minutes, instead of every 'tenth minute'

### ISSUE THREAD

**Issue by domruf** [allinurl/goaccess#1251]
Currently the time distribution is either 1 hour or 10 minutes.
It would be great if there were an option to use 1 minute as well.
Or even better use a number value to use 1, 2, 3, 4, etc. minutes.

related to #360

**Comment by supportmaster** [comment]
Yes, looking forward to this feature.  10 minutes is nice, but every 1 minute is really what I need.

**Comment by hnzufx** [comment]
This would be a nice feature, especially when you can specify it by seconds. This way you can use goaccess to monitor log files live (apache, nginx and varnish).
